lib/index.js

```diff
@@ -0,0 +1,80 @@
+'use babel';
+
+import path from 'path';
```

Comment:
Just a side note that this will break in the future versions of node because they don't export a `default` for you to use here, either do
```
import * as path from 'path'
```js
or do
```
import {dirname as getDirNameFromPath} from 'path'
```js
^ add anything you wanna import there